test/standalone/Soil/climate_drivers.jl

```diff
@@ -145,8 +148,8 @@ for FT in (Float32, Float64)
             init_soil!(Y, coords.subsurface.z, model.parameters)
             set_initial_aux_state! = make_set_initial_aux_state(model)
             set_initial_aux_state!(p, Y, t)
-
-
+            tendency! = make_exp_tendency(model)
+            tendency!(similar(Y), Y, p, t)
```

Comment:
why do we need to call `tendency!` here now? I would expect `p` to get set with the initial `Y` when we call `set_initial_aux_state!(p, Y, t)`, so then calling `tendency!(...)` and e.g. `R_n = ClimaLSM.net_radiation(...)` would make `R_n` one timestep ahead of `p.soil.R_n`

Reply:
yes, there is something not great going on! We arent actually stepping the state `Y` when we call tendency!, just computing `dY`, and the `dY` is used to increment `Y` in a separate step. so `p` and `Y` should still be in sync after calling `tendency!`. I think the issue is that we have some aux variables that are updated in `update_aux!` and some that are updated in the tendency. 

it would be great if we had a way around this.  for standalone models I think we could move things so that the boundary fluxes are computed in `update_aux!`, and then just used in the tendency function. (The boundary fluxes depending on R_n, SHF, LHF, etc). But with integrated models, things get more complex, because canopy aux variables can affect the boundary conditions for the soil, and vice versa. it's worth more thought though. because `update_aux!` is becoming more misleading as a name. Also, the forcing variables are in the cache too, but will get update first in a separate function (at least in the idea we had discussed earlier).

Reply:
I vaguely wonder about:
```
function update_aux!(p,Y,t)
    update_forcing!(p,Y,t)
    update_standalone_aux!(p,Y,t) # our current update aux
    update_boundary_conditions!(p,Y,t)
end
```
or something? TBD 😵‍💫

Reply:
we could move towards atmos's naming style and use `update_cache_vars!` instead of `update_aux!`? though maybe it would be confusing if our "cache" means something slightly different from theirs